hack/lib/install.sh

```diff
@@ -33,6 +33,10 @@ function check_kind {
   if [[ $? -ne 0 ]]; then
     echo "installing kind ."
     GO111MODULE="on" go get sigs.k8s.io/kind@v0.5.1
+    if [[ $? -ne 0 ]]; then
```

Comment:
This if-clause is used to test whether kind is installed successfully :)

Reply:
yes I found my fault